scripts/lib/website-comparisons.ts

```diff
@@ -6,6 +6,10 @@ export function cleanWebsiteData(response: string): string {
   const $ = cheerio.load(response);
   // Only check for the HTML body to try and avoid metadata and scripts that may change per-system/call.
   if ($('body').length > 0) {
+    // If <p> tags exist, try to only return those, as they'll likely be relevant tags.
+    if ($('p').length > 0) {
+      return $('p').contents().text().trim();
```

Comment:
I don't have an intuition about what this looks like if there are multiple <p> tags. One option is for me to just not worry about it as long as it works, since this text is hashed anyway. But if you think there is a comment that might succinctly describe what the output of this line is (is it concatenated together? Is it an array?), that might help.

Reply:
There sure is! Added comment.